articles/libraries/lock-android/native-provider-facebook.md

```diff
@@ -147,68 +163,13 @@ provider.clearSession();
 ```
 
 ### Remember the Last Login
-By default, this provider will remember the last account used to log in. If you want to change this behavior, use the following method.
+By default this provider will remember the last account used to log in. If you want to change this behavior, use the following method.
 
 ```java
 provider.rememberLastLogin(false);
 ```
 
 
-## Certificate Fingerprints
-When creating a new App in the Facebook Developers Console, you will need to provide the SHA-1 of the certificate you're using to sign your application in Base64 encoding. To obtain it follow this steps:
-
-Locate the certificate you're using to sign your application. If you don't have one you can generate it or use the default `android.keystore` certificate, that was generated automatically for you when you installed the SDK. In this example we're going to use the default one.
-
-* On Windows:
-
-```sh
-keytool -exportcert -alias androiddebugkey -keystore %USERPROFILE%\.android\debug.keystore -storepass android | openssl sha1 -binary | openssl base64
-```
-
-* On Linux / macOS:
-
-```sh
-keytool -exportcert -alias androiddebugkey -keystore ~/.android/debug.keystore -storepass android | openssl sha1 -binary | openssl base64
```

Comment:
Keep this, maybe we can avoid the windows/mac diff snippet by just saying "path-to-the-keystore"